crates/pina_cli/src/init.rs

```
@@ -71,38 +72,46 @@ pub fn init_project(project_dir: &Path, package_name: &str, force: bool) -> Resu
 	println!("    cd {}", project_dir.display());
-	println!("    cargo build                    # build the library");
-	println!("    cargo build --release \\",);
-	println!("      --target bpfel-unknown-none \\",);
-	println!("      -p {package_name} \\",);
-	println!("      -Z build-std -F bpf-entrypoint  # build for SBF",);
+	println!("    pina build                     # build SBF and generate the IDL");
 	println!("    cargo test                     # run tests");
-	println!("    pina idl --path .              # generate Codama IDL JSON");
+	println!("    pina generate                  # generate configured clients");
 	println!();
 	println!("  Update the program address in src/lib.rs with your deployed program ID.",);
```

Clippy (1.97.1)

[warning] unnecessary trailing comma

(warning)